.github/workflows/build.yml

```diff
@@ -206,7 +206,6 @@ jobs:
           path: coverage.xml
 
   sonarcloud:
-    needs: [test]
     name: SonarCloud
     runs-on: ubuntu-20.04
     steps:
```

Comment:
Will need to make a change to the `download-artifact` job to make sure a fail here doesn't prevent a red Error to show up in the pipeline:
https://github.com/Sage-Bionetworks/synapsePythonClient/actions/runs/9022604848/job/24792505985?pr=1094
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/c023edf9-1c76-4335-890d-bc2cd18a41a3)
You'll probably need a step similar to:
https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/.github/workflows/build.yml#L113-L121

That runs a bash script to check if the file is present. If not present don't run `Download coverage report` or `Override Coverage Source Path for Sonar`

Reply:
Yea, I forgot to add this and will work on it soon. Thanks~